app/lib/io/zipUtil.js

```diff
@@ -12,7 +12,26 @@ define([
     var NotImplementedError = errors.NotImplemented;
 
     var unpack = function(async, zipData, io, targetPath){
-        throw new NotImplementedError('ZIP unpack method is not yet implemented');
+        if (async)
+            throw new NotImplementedError('Asynchronous ZIP unpack method is not yet implemented');
+
+        var zip = new JSZip(zipData);
+        var files = zip.files;
+        console.log("unpacking files: ", files);
+        var i;
+
+        for (i in files){
```

Comment:
if `files` is an array, this is not the right way to iterate over it.
If files is an object `for (k in files)` would clarify you intention.
However, since you don't have a further use for `i` (instead you need to do `file.name`)I believe this is an array.